doc/dev/release_checklist.md

```diff
@@ -202,6 +206,8 @@ We'll use `v4.7.0-rc1` as the intended release version in this example.
   - It is essential for Mathlib and Batteries CI that you then create the next `bump/v4.8.0` branch
     for the next development cycle.
     Set the `lean-toolchain` file on this branch to same `nightly` you used for this release.
+  - Deploy the reference manual for the release candidate to the URL specified above. It will be replaced by
+    the final release's manual as part of the release process.
```

Comment:
Similarly here. What do I need to do?

Reply:
Pushing the tag will now do it automatically. Thanks!